### DISCUSSION TITLE
Does Naubot has export option in Job Result page?

### DISCUSSION THREAD

**Discussion by alicechiou728** [nautobot/nautobot#6092] [Q&A] [answered]
Does Naubot has export option in Job Result page? 
If we executed same jobs on multiple devices, Is there any export function that we can export the job results then analysis the data?
![image](https://github.com/user-attachments/assets/b42fe3a4-a049-47e4-a520-9011c2e59b34)

**Comment by glennmatthews** [comment]
You can export the logs from a given JobResult from its detail view:

![image](https://github.com/user-attachments/assets/a77111ae-0c0b-4b5b-913e-ceb0941cc1ec)

**Reply by alicechiou728** [reply]
This is for single job result, but what we want is check multi job results.

**Comment by joewesch** [comment, marked as answer]
Depending on what you are trying to extract, you could use a graphql query. Here is a query to get you started that gets all of the log messages for a particular job model:

```graphql
query {
  job_results(job_model: "Your Job Name") {
    job_log_entries {
      message
    }
  }
}
```

**Reply by alicechiou728** [reply]
Thanks, it works. By the way, is it possible a future version add a button to export job results list as a new function?

**Reply by joewesch** [reply]
I don't see any specific feature requests about exposing the JobLogEntry objects as a list (table) view that can be filtered or exported, but feel free to add one if you'd like and the core team can take it under advisement.
